### PR TITLE
reorganize text and expand on the solution

### PR DESCRIPTION
**Pull request by adrienpoly**
Hey 

I started to comment on the initial PR but found it easier finally to open a new PR on yours as I wanted to suggest several significant changes.

- I expanded the solution description and mostly explaining more in detail the role of the newly defined `errorHandler` function within the `ApplicationController`

- I removed most of this part:

> Stimulus includes a largely undocumented global error handler (see links below) that catches errors when Stimulus invokes your application code in internal try/catch blocks.

as it is not as magic as that but only a result of the newly defined `errorHandler` function in the `ApplicationController`

- move to a dedicated section the configuration of the reporting service

Basically the idea is lets make sure we collect all the issues in the same place and format them correctly. Then once we have this single point let's dispatch this source to a dedicated service.

Feel free to comment/edit etc

### CODE REVIEW

**Comment by julianrubisch** [comment]
This is such great feedback, thanks!

File: error_handling/global-error-handler.md [hunk 1]
```diff
@@ -6,37 +6,28 @@ parent: Error Handling
...
+First define a [global Application Controller](./architecture/application-controller.md) that your Stimulus controllers will inherit from.
 
-// configure Sentry to log errors and prepend the default handler
-const sentryErrorHandler = (error, message, detail = {}) => {
-  defaultErrorHandler(error, message, detail)
-  Sentry.captureException(error, { message, ...detail })
-}
+Within this ApplicationController define a new `handleError` function. Here this function creates a small context object with the id of the current user (you can pass all context information that can help you debug: environment, subscription status, etc).
+
+Stimulus includes a largely undocumented error handler (see links below). By default, this error handle catches all Stimulus internal errors and output a prettify `console.log`.
```
Comment (julianrubisch) [inline error_handling/global-error-handler.md:22]: "prettified"

Comment (adrienpoly) [reply error_handling/global-error-handler.md]: thanks for picking it
